FAILED src/integrity_tests/test_cudo.py::test_locations

`Test catalog integrity` failed today.
https://github.com/dstackai/gpuhunt/actions/runs/8105089998/job/22172772003

The servers located in the 'se-smedjebacken-1' location can sometimes be unavailable.

```
Run pytest src/integrity_tests
============================= test session starts ==============================
platform linux -- Python 3.11.8, pytest-8.0.2, pluggy-1.4.0
rootdir: /home/runner/work/gpuhunt/gpuhunt
collected 25 items

src/integrity_tests/test_all.py F                                        [  4%]
src/integrity_tests/test_aws.py ...                                      [ 16%]
src/integrity_tests/test_azure.py .....                                  [ 36%]
src/integrity_tests/test_cudo.py F..                                     [ 48%]
src/integrity_tests/test_datacrunch.py ....                              [ 64%]
src/integrity_tests/test_gcp.py .....                                    [ 84%]
src/integrity_tests/test_nebius.py ....                                  [100%]

=================================== FAILURES ===================================
______________________ TestAllCatalogs.test_non_zero_cost ______________________

self = <integrity_tests.test_all.TestAllCatalogs object at 0x7f5729aaaa90>
catalog_files = [PosixPath('lambdalabs.csv'), PosixPath('cudo.csv'), PosixPath('nebius.csv'), PosixPath('gcp.csv'), PosixPath('aws.csv'), PosixPath('datacrunch.csv'), ...]

    def test_non_zero_cost(self, catalog_files: List[Path]):
        for file in catalog_files:
            with open(file, "r") as f:
                reader = csv.DictReader(f)
                prices = [float(row["price"]) for row in reader]
>           assert 0 not in prices
E           assert 0 not in [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, ...]

src/integrity_tests/test_all.py:19: AssertionError
________________________________ test_locations ________________________________

data_rows = [{'cpu': '1', 'disk_size': '', 'gpu_count': '1', 'gpu_memory': '16', ...}, {'cpu': '1', 'disk_size': '', 'gpu_count': ...u_count': '1', 'gpu_memory': '16', ...}, {'cpu': '3', 'disk_size': '', 'gpu_count': '1', 'gpu_memory': '16', ...}, ...]

    def test_locations(data_rows):
        expected = {
            "no-luster-1",
            "se-smedjebacken-1",
            "se-stockholm-1",
            "us-newyork-1",
            "us-santaclara-1",
        }
        locations = select_row(data_rows, "location")
>       assert set(locations) == expected
E       AssertionError: assert {'no-luster-1...santaclara-1'} == {'no-luster-1...santaclara-1'}
E         
E         Extra items in the right set:
E         'se-smedjebacken-1'
E         
E         Full diff:
E           {
E               'no-luster-1',
E         -     'se-smedjebacken-1',
E               'se-stockholm-1',
E               'us-newyork-1',
E               'us-santaclara-1',
E           }

src/integrity_tests/test_cudo.py:32: AssertionError
---------------------------- Captured stdout setup -----------------------------
.
=========================== short test summary info ============================
FAILED src/integrity_tests/test_all.py::TestAllCatalogs::test_non_zero_cost - assert 0 not in [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, ...]
FAILED src/integrity_tests/test_cudo.py::test_locations - AssertionError: assert {'no-luster-1...santaclara-1'} == {'no-luster-1...santaclara-1'}
  
  Extra items in the right set:
  'se-smedjebacken-1'
  
  Full diff:
    {
        'no-luster-1',
  -     'se-smedjebacken-1',
        'se-stockholm-1',
        'us-newyork-1',
        'us-santaclara-1',
    }
========================= 2 failed, 23 passed in 0.35s =========================
Error: Process completed with exit code 1.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -12,6 +12,6 @@ repos:
         name: isort (python)
         args: ['--settings-file', 'pyconfig.toml']
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.2.2
+    rev: v0.3.0
     hooks:
       - id: ruff

--- a/src/gpuhunt/providers/cudo.py
+++ b/src/gpuhunt/providers/cudo.py
@@ -15,6 +15,10 @@ logger = logging.getLogger(__name__)
 
 API_URL = "https://rest.compute.cudo.org/v1"
 
+SKIP_DC = [
+    "se-smedjebacken-1",  # sometimes there are no available VMs
+]
+
 
 class CudoProvider(AbstractProvider):
     NAME = "cudo"
@@ -52,9 +56,12 @@ class CudoProvider(AbstractProvider):
             if gpu and name is None:
                 logger.warning("Skip. Unknown GPU name: %s", vm["gpuModel"])
                 continue
+            data_center = vm["dataCenterId"]
+            if data_center in SKIP_DC:
+                continue
             raw = RawCatalogItem(
                 instance_name=vm["machineType"],
-                location=vm["dataCenterId"],
+                location=data_center,
                 spot=False,
                 price=round(float(vm["totalPriceHr"]["value"]), 5),
                 cpu=vcpu,

--- a/src/integrity_tests/test_cudo.py
+++ b/src/integrity_tests/test_cudo.py
@@ -23,7 +23,6 @@ def select_row(rows, name: str) -> List[str]:
 def test_locations(data_rows):
     expected = {
         "no-luster-1",
-        "se-smedjebacken-1",
         "se-stockholm-1",
         "us-newyork-1",
         "us-santaclara-1",

